### ISSUE TITLE
multiworld time synchronization?

### ISSUE THREAD

**Issue by meoso** [Multiverse/Multiverse-Inventories#269]
maybe already requested, but can there be a multiworld timesync command or config for continuous sync?

**Comment by luckydonald** [comment]
This is not quite inventory related, but this plugin is already providing grouping of worlds.
Would be great.

**Comment by dumptruckman** [comment]
Yeah.. not sure if Inventories should be doing this. However, it does group worlds so in a way it seems like it would be the place for it. If I ever revive this plugin it is something I could consider. However, it also seems like a feature that would be prone to bugs.
